input/community/speakers/hamida-rebai.md

```diff
@@ -0,0 +1,49 @@
+Title: Hamida Rebai
+Location: Quebec Canada
+Email: didourebai@gmail.com
+Language:
+  - English
+  - french
+  - arabic
+Topics:
+  - .NET
+  - ASP.NET
+  - ASP.NET MVC
+  - Architecture
+  - Azure
+  - C#
+  - Containers
+  - DevOps
+  - Diversity & Inclusion
+  - Entity Framework
+  - Microsoft 365
+  - Microsoft Teams
+  - Microservices
+  - ML.NET
+  - Mobile Development
+  - NuGet
+  - Razor
+  - SignalR
+  - Visual Studio
+  - Visual Studio Code
+  - Visual Studio for Mac
+  - Web Development
+  - Windows Development
+  - Windows Forms
+  - Xamarin
+  - Xamarin.Forms
+Blog: https://medium.com/@didourebai
```

Comment:
```suggestion
Blog: "https://medium.com/@didourebai"
```

Wrapping this in quotes because I think I saw that the parsing was breaking on `@` symbols in strings. Figured it can't hurt.